README.md

```diff
@@ -292,6 +292,44 @@ end
 
 The application will be an instance of your application. callback_uri is a url with which you want to handle the ID Site information, this url also needs to be set in the Stormpath’s dashboard on [ID Site settings page](https://api.stormpath.com/ui2/index.html#/id-site) as Authorized Redirect URLs.
 
+##### Using ID Site for [multitenancy][id-site-multitenancy]
+
+When a user wants to login to your application, you may want to specify an organization for the user to login to. Stormpath ID Site is configurable to support multitenancy with Organization resources
+
+```ruby
+application.create_id_site_url({
+  callback_uri: 'https://trooperapp.com/callback',
+  organization_name_key: 'stormtrooper',
+  show_organization_field: true
+});
+```
+
+##### Using Subdomains 
+
+In some cases, you may want to show the organization that the user is logging into as a subdomain instead of an form field. To configuring this, you need to use a [wildcard certificate][wildcard-certificate] when setting up your [custom domain with ID Site][custom-domain-with-id-site]. Otherwise, the Stormpath infrastructure will cause browser SSL errors.
```

Comment:
Minor grammar correction here: please change "To configuring this" to "To configure this".